src/cryo_tank/tanktools.jl

```diff
@@ -1,24 +1,22 @@
 """
-    find_mdot_time(t, pari, parg, para, pare)
+    find_mdot_time(t, parg, para, pare)
 
 This function outputs the fuel mass flow rate to the engines as a function of time for a TASOPT aircraft 
 model.
 !!! details "🔃 Inputs and Outputs"
     **Inputs:**
     - `t::Float64`: time in mission (s)
-    - `parg::Vector{Float64}`: vector with aircraft integer parameters
     - `parg::Vector{Float64}`: vector with aircraft geometric parameters
     - `para::Array{Float64}`: array with aircraft aerodynamic parameters
     - `pare::Array{Float64}`: array with aircraft engine parameters
     
     **Outputs:**
     - `t::mdot`: fuel mass flow rate out of the tank (kg/s)
 """
-function find_mdot_time(t::Float64, pari::Vector{Int64}, parg::Vector{Float64}, para::Array{Float64}, pare::Array{Float64})
-    nftanks = pari[iinftanks]
+function find_mdot_time(t::Float64, fuselage_fueltank_count::Int64, parg::Vector{Float64}, para::Array{Float64}, pare::Array{Float64})
```

Comment:
```suggestion
function find_mdot_time(t::Float64, fueltank_count::Int64, parg::Vector{Float64}, para::Array{Float64}, pare::Array{Float64})
```
Probs can simplify by dropping "fuselage" since rn we are not going to have situations where we have multiple non-fuselage fuel tanks (which for now is just the wing)

Reply:
that's true. that said, i had the thought at some point "does nftanks include the wing? are we counting the wingbox as separate?" silly when you know how it works, less so before

Reply:
that said, this function rn is only applicable to fuse tanks. done for brevity